src/ontology/cl-edit.owl

```diff
@@ -27234,7 +27249,9 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_1001225 obo:CL_1000497
 
 # Class: obo:CL_1001285 (vasa recta descending limb cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20392798"^^xsd:string) obo:IAO_0000115 obo:CL_1001285 "A cell that is part of the vasa recta descending limb. It is known in some mammalian species that this cell may express the urea transporter B.")
```

Comment:
Same as above.

Reply:
I have moved it to editor_comment and added more references